public/app/features/plugins/sql/datasource/SqlDatasource.ts

```diff
@@ -164,7 +164,7 @@ export abstract class SqlDatasource extends DataSourceWithBackend<SQLQuery, SQLO
         .pipe(
           map((res: FetchResponse<BackendDataSourceResponse>) => {
             const rsp = toDataQueryResponse(res, queries);
-            return rsp.data[0];
```

Comment:
This would still cause an error in
https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/mssql/datasource.ts#L26

Reply:
Good catch! Thank you @zoltanbedi !